Frequently Asked Questions
___________________________________

*What is DBiT-seq: Deterministic Barcoding in Tissue for spatial omics sequencing?*

Deterministic barcoding is a method used in spatial omics sequencing to tag individual cells or tissue samples with unique barcodes. This allows researchers to track the cells or samples throughout the sequencing process, and to accurately assemble the resulting data into a single, comprehensive map of the genome or transcriptome.
In spatial omics sequencing, deterministic barcoding is typically performed using microfluidic or other high-throughput methods, which allow many cells or samples to be processed simultaneously. The barcodes are attached to the cells or samples using specific oligonucleotides or other molecules that are complementary to the target DNA or RNA. This ensures that the barcodes are accurately and reliably attached to the cells or samples, and that they do not interfere with the subsequent sequencing reactions.
Deterministic barcoding is a critical step in spatial omics sequencing, as it allows researchers to accurately and reliably track the cells or samples throughout the process and to generate high-quality, comprehensive data sets. This information can be used to study cellular development, differentiation, and disease at a high resolution, providing valuable insights into the complex mechanisms underlying these processes.

*What is a Tixel?*

A Tixel is region on the tissue that has had both barcodes from the B and A chip. Tixels are the basis of how spatial information is encoded from the DBiT process.

*What is a Unique Molecular Identifier (UMI)?*

A unique molecular identifier (UMI) is a short, arbitrary sequence of DNA or RNA that is used to tag individual molecules or samples in a biological experiment. UMIs are typically used in high-throughput sequencing experiments, where they can help to identify and distinguish between multiple molecules or samples that have been sequenced together.
UMIs are often used in conjunction with other techniques, such as barcoding, to provide additional information about the molecules or samples that are being studied. For example, a UMI might be combined with a barcode that identifies the specific cell or tissue sample that the molecule came from, allowing researchers to track the molecule through the sequencing process and assemble the data into a comprehensive map of the genome or transcriptome.
UMIs are useful because they are unique, arbitrary sequences that can be easily generated and attached to individual molecules or samples. This allows us to accurately and reliably track the molecules throughout the experiment, and to generate high-quality, comprehensive data sets that can be used to study cellular development, differentiation, and disease.

*What is the Region of Interest (ROI)?*

Region of Interest (ROI) is a specific area of the genome that is of particular interest or relevance to the researcher. This might be a region known to contain a gene of interest, a cluster of regulatory elements, or any other region that is of interest for the specific research question being addressed.
Defining and analyzing ROIs is an important part of ATAC-seq and dBIT sequencing, as it allows us to focus our analysis on specific areas of the genome that are most likely to provide valuable insights. This can improve the overall efficiency and effectiveness of the analysis.

*What is Spatial ATAC-seq?*

Spatial ATAC-seq (Assay for Transposase-Accessible Chromatin using sequencing) is a high-throughput, genome-wide method for mapping the locations of open chromatin in cells. The technique uses transposase, an enzyme that can insert itself into the genome at specific sites, to selectively cut open the chromatin at these sites and make the DNA accessible for sequencing. The resulting data provides a detailed map of the open chromatin in the cell, which can be used to identify regulatory elements and study gene expression and regulation.
Spatial ATAC-seq is an advanced variation of the traditional ATAC-seq technique, which uses bulk tissue samples rather than individual cells. By analyzing the open chromatin at the single-cell level, spatial ATAC-seq allows researchers to study the spatial organization of the genome and the variations in gene expression and regulation between different cells. This information can be used to better understand cellular development, differentiation, and disease.

*What is a Gene Score?*

A gene score is a numerical value that is assigned to a gene based on certain characteristics or criteria. This might include the level of transcription factor binding, the level of open chromatin, or other features that are relevant to the specific research question being addressed.
Gene scores are often used in the analysis of ATAC-seq and dBIT data to identify genes that are particularly active or important within the cell or tissue being studied. By calculating gene scores based on the data generated by these techniques, researchers can prioritize which genes to study in more detail, and can identify potential targets for further investigation.
Overall, gene scores are a valuable tool in the analysis of ATAC-seq and dBIT data, as they provide a way to quantitatively compare genes and to identify those that are likely to be relevant to the research question being addressed. By using gene scores, researchers can focus their efforts on the most promising areas of the genome, and can generate more meaningful and informative results from their experiments.

*What is the difference between the motif and gene dataset?*

A motif dataset is a collection of data that provides information about the sequence motifs that are present within a genome. A sequence motif is a short, recurring pattern of nucleotides that is found within the DNA or RNA of a genome, and is often associated with specific functions or regulatory roles. A motif dataset typically includes information about the locations, frequencies, and patterns of these motifs within the genome, as well as any other characteristics that are relevant to the specific research question being addressed.
A gene dataset, on the other hand, is a collection of data that provides information about the genes within a genome, such as their locations, sequences, and expression levels. This type of dataset typically includes information about the genes' coding regions and regulatory elements, as well as any other features that are relevant to the specific research question being addressed.
In AtlasXplore, both motif and gene datasets can be visualized and explored using the tool's user-friendly interface. Users can view the data and zoom in and out to focus on specific regions of interest, and can use the tool's various features and tools to highlight and annotate interesting patterns or features within the data. By combining and comparing motif and gene datasets, researchers can gain a more comprehensive understanding of the organization and regulation of the genome, and can identify potential targets for further investigation.
